ecosystem/sep-0012.md

```diff
@@ -98,15 +98,15 @@ GET [KYC_SERVER]/customer?id=<customer-id>&type=<customer-type>
 Name | Type | Description
 -----|------|------------
 `id` | `string` | (optional) The ID of the customer as returned in the response of a previous `PUT` request. If the customer has not been registered, they do not yet have an `id`.
-`account` | `G...` or `M...` string | (**deprecated**, optional) This field should match the `sub` value of the decoded SEP-10 JWT.
+`account` | `G...` or `M...` string | (**deprecated**, optional) This field should match the `sub` value of the decoded SEP-10 JWT. When omitted, the server should use the `sub` value of the decoded SEP-10 JWT.
```

Comment:
Maybe we should replace the existing description entirely, so its clear that omitting the field is the preferred approach. This works though.

Reply:
Addressed in https://github.com/stellar/stellar-protocol/pull/1302/commits/fd40428111ccd66e77f36ebefcbdaf087d2e9ea8.